### ISSUE TITLE
NPE when creating a new file in src/main using intellij Build #IU-233.11799.241

### ISSUE THREAD

**Issue by jonas-grgt** [sweet-mustard/test-nurturer#30]
When creating a new java file I get the following exception, my file is not created.

```
GenerateTestMotherAction#Update@ProjectViewPopup (be.sweetmustard.testnurturer.GenerateTestMotherAction), actionId=be.sweetmustard.testnurturer.GenerateTestMotherAction, text='Generate Test Mother'

java.lang.NullPointerException: Cannot invoke "com.intellij.psi.PsiFile.getName()" because the return value of "com.intellij.psi.PsiElement.getContainingFile()" is null
	at be.sweetmustard.testnurturer.GenerateTestMotherAction.update(GenerateTestMotherAction.kt:55)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performDumbAwareUpdate$0(ActionUtil.java:164)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:187)
	at com.intellij.openapi.actionSystem.impl.ActionUpdaterKt.doUpdate(ActionUpdater.kt:696)
	at com.intellij.openapi.actionSystem.impl.ActionUpdaterKt.access$doUpdate(ActionUpdater.kt:1)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateActionReal$success$1.invoke(ActionUpdater.kt:110)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$updateActionReal$success$1.invoke(ActionUpdater.kt:109)
	at com.intellij.openapi.actionSystem.impl.ActionUpdater$callAction$3$adjustedCall$1.invoke(ActionUpdater.kt:157)
	at com.intellij.openapi.application.rw.InternalReadAction.insideReadAction(InternalReadAction.kt:108)
	at com.intellij.openapi.application.rw.InternalReadAction.access$insideReadAction(InternalReadAction.kt:16)
	at com.intellij.openapi.application.rw.InternalReadAction$tryReadCancellable$2.invoke(InternalReadAction.kt:95)
	at com.intellij.openapi.application.rw.InternalReadAction$tryReadCancellable$2.invoke(InternalReadAction.kt:94)
```

**Comment by wimdeblauwe** [comment]
This could be a duplicate of #19. Would be good to retest once I have a release out with that fix.

**Comment by wimdeblauwe** [comment]
@jonasgeiregat Can you test with the just released 0.1.0 version?

**Comment by wimdeblauwe** [comment]
Closing as there has been no further feedback. Please re-open if you would still have the issue.
